Refatoração e padronização dos scripts de movimento

jpholanda opened the pull request:
Padronizados nomes de variáveis, propriedades e métodos para CamelCase (variáveis iniciam com minúsculas, os outros maiúsculas). Removidos elementos inutilizados. Alguns trechos de código simplificados. Adicionados comentários e documentação para cada variável, propriedade e método.

JohnVithor commented:
Gostei bastante das modificações e otimizações feitas. Principalmente da adição dos comentários. 
Tentarei seguir esse padrão agora.
Por mim está tudo ok com esse pull request.

Commit:
Refatoração e padronização dos scripts de movimento

Nomes de variáveis, propriedades e métodos padronizados para CamelCase (variáveis iniciam com minúsculas, os outros maiúsculas). Elementos que não serão utilizados foram removidos. Alguns trechos de código foram simplificados. Documentação adicionada para cada variável, propriedade e métodos não-Unity.

Commit:
Configuração do script de movimento do player e do prefab Enemy

Valor dos parâmetros de teclas de entrada para movimento do player alterado para manter o comportamento atual. Duração do movimento do player na cena de teste e do prefab Enemy alteradas.

jpholanda commented:
@giordanorn Separei a alteração do código e da cena em dois commits. Revise, por favor, e aprove se tiver tudo certo.

Commit:
Reorganização da pasta Scenes

Separação de cenas de teste das cenas principais em subpastas da pasta Scenes.

giordanorn committed:
Branches atualizadas
